dev_guide/cicd.adoc

```diff
@@ -0,0 +1,254 @@
+[[dev-guide-cicd]]
+= CI / CD in OpenShift
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+{product-title} This next topic will focus on continuous integration and continuous delivery (`*CI/CD*`) of applications with OpenShift.  In particular, the goal is to
```

Comment:
ah - that did not occur to me, but yeah, in hindsight makes sense.

OK, I'll globally replace OpenShift and OpenShift Origin with {product-title}.  Assuming that even includes line 2.